polkadot/xcm/src/v3/multiasset.rs

```diff
@@ -506,8 +506,7 @@ impl TryFrom<OldMultiAsset> for MultiAsset {
 #[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
 pub struct MultiAssets(Vec<MultiAsset>);
 
-/// Maximum number of items we expect in a single `MultiAssets` value. Note this is not (yet)
-/// enforced, and just serves to provide a sensible `max_encoded_len` for `MultiAssets`.
+/// Maximum number of items in a single `MultiAssets` value that can be decoded.
 const MAX_ITEMS_IN_MULTIASSETS: usize = 20;
```

Comment:
I would always make this stuff public, otherwise downstream crates will be confused why an opaque number is used.